README.md

```diff
@@ -68,4 +68,21 @@ To run the tests for the codebase run the following command:
 ```
 poetry run pytest
 ```
-Please Note: Make sure you have installed `pytest` by running `poetry add pytest` before running the tests 
\ No newline at end of file
+Please Note: Make sure you have installed `pytest` by running `poetry add pytest` before running the tests 
+
+## Building and Running the App via Docker
+
+To build the container for local development:
+```bash
+docker build --target development --tag todo-app:dev .
+```
+
+To run the container for local development:
+```bash
+docker run --publish 8000:5000 -it --env-file .env --mount "type=bind,source=$(pwd)/todo_app,target=/app/todo_app" todo-app:dev 
+```
+For the production container, the build and run commands are:
+```bash
+docker build --target production --tag todo-app:prod .
+docker run --publish 8000:5000 -it --env-file .env todo-app:prod
```

Comment:
Thanks for the clear instructions, these made it very easy to test everything and it all _just worked_ 😄